thamos/lib.py

```diff
@@ -255,6 +254,39 @@ def advise(
 
     return response.result, response.result["error"]
 
+def advise_here(
+    recommendation_type: str = None,
+    *,
+    runtime_environment: dict = None,
+    runtime_environment_name: str = None,
+    limit_latest_versions: int = None,
+    no_static_analysis: bool = False,
+    nowait: bool = False,
+    force: bool = False,
+    limit: int = None,
+    count: int = 1,
+    debug: bool = False,
+) -> typing.Optional[tuple]:
+    '''Run advise in current directory, requires no arguments'''
+    if not os.path.isfile("Pipfile"):
+        raise FileNotFoundError("No Pipfile found in current directory")
+    
+    with open("Pipfile", "r") as pipfile, \
+         open("Pipfile.lock", "a+") as piplock:
```

Comment:
Why does it create Pipfile.lock?

Reply:
I want to read in Pipfile.lock if it exists and otherwise just pass an empty string.  It gets rid of the need to check if the file exists while keeping the same behaviour.

Reply:
I would suggest keeping the filesystem untouched in this case. If the analysis errors out, there will be present empty Pipfile.lock which is not valid Pipfile.lock file. You are reading whole content anyway, so its just 2 more lines in sources.

Reply:
Okay